src/utils/time.ts

```diff
@@ -263,41 +263,31 @@ export function nextNightAtLocation(
         return date.getTime() + getDuration({days: 1});
     }
 
-    const sunriseTime = time.sunriseTime;
-    const sunsetTime = time.sunsetTime;
+    const [time1, time2] = time.sunriseTime < time.sunsetTime ? [time.sunriseTime, time.sunsetTime] : [time.sunsetTime, time.sunriseTime];
```

Comment:
Same thing, could be more verbose, e.g. FirstTimeOnDay or something in that way that indicates it's the lowest time of the 2 times.